pages/linux/fossil-add.md

```diff
@@ -0,0 +1,12 @@
+# fossil add
+
+> Put files or directories under Fossil version control.
+> More information: <https://fossil-scm.org/home/help/add>.
+
+- Stage a file or directory for the next commit to the current checkout:
```

Comment:
This makes it sound like it works like git. How about this?
```suggestion
- Put a file or directory under version control, so it will be in the current checkout:
```
Or similar. But `stage a file` sounds like it is already under version control.

Reply:
Excellent point, thanks!